apps/site/src/app/(home)/sections/Clubs/Clubs.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @next/next/no-img-element */
+import { getClubs } from "./getClubs";
+import styles from "./Clubs.module.scss";
+import { client } from "@/lib/sanity/client";
+import imageUrlBuilder from "@sanity/image-url";
+import Container from "react-bootstrap/Container";
```

Comment:
Chore: rearrange import order to be in the order of library components, our custom components, assets, then styles